File: 3318/3318.go
func findXsumOfAllKlongSubarraysI(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findXsumOfAllKlongSubarraysI(root.Left)
	r := findXsumOfAllKlongSubarraysI(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 3318/3318.cpp
class Solution {
public:
    int findXsumOfAllKlongSubarraysI(TreeNode* root) {
        if(!root) return 0;
        return 1+max(findXsumOfAllKlongSubarraysI(root->left),findXsumOfAllKlongSubarraysI(root->right));
    }
};